task2/rest-api-basics/repository/src/main/java/com/epam/esm/repository/impl/mapper/TagMapper.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.esm.repository.impl.mapper;
+
+import com.epam.esm.entity.Tag;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Component;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+@Component
+public class TagMapper implements RowMapper<Tag> {
+
+    public static final String ID = "t_id";
+    public static final String NAME = "t_name";
+
+    public static final String SECONDARY_ID = "id";
```

Comment:
Delete unused fields everywhere

Reply:
> Delete unused fields everywhere

But it is used in https://github.com/igor-zhuravlev/mjc/blob/c9cb85a679e210b42e235310c95177045a8b78c6/task2/rest-api-basics/repository/src/main/java/com/epam/esm/repository/impl/TagRepositoryImpl.java#L74 when I recieve a generated id

Reply:
I didn't notice. Resolved

Reply:
> I didn't notice. Resolved

Sorry for strange question, but how I can resolve other conversations? I need a commented their as fixed or maybe simply click resolve

Reply:
just resolve them